nevergrad/optimization/optimizerlib.py

```diff
@@ -2322,85 +2322,6 @@ class NGO(NGOptBase):  # compatibility
     pass
 
 
-@registry.register
-class NGOpt2(NGOptBase):
```

Comment:
why is it disappearing? happy that it does anyway :D

Reply:
Not used anymore.
The main issue for removing old stuff is the interference with mixsimulator; there are many things to be removed.

Reply:
it's probably preferable to remove it anyway, if it should not be used.
But breaking other people's code is always a bit tricky :s that's why I prefer avoiding creating too many different registered optimizers :D